frontends/web/src/components/layout/header.tsx

```diff
@@ -38,11 +39,11 @@ class Header extends Component<Props> {
   };
 
   public render() {
-    const { t, title, narrow, children, guideExists, shown, sidebarStatus } = this.props;
+    const { t, title, narrow, children, guideExists, shown, sidebarStatus, hideSidebarToggler } = this.props;
     return (
       <div className={[style.container, sidebarStatus ? style[sidebarStatus] : ''].join(' ')}>
         <div className={[style.header, narrow ? style.narrow : ''].join(' ')}>
-          <div className={style.sidebarToggler} onClick={toggleSidebar}>
+          <div className={`${style.sidebarToggler} ${hideSidebarToggler ? style.hideSidebarToggler : ''}`} onClick={toggleSidebar}>
```

Comment:
Just made a change to hide this via CSS so we can hide it at the correct media query / screen size. Previously we're hiding the toggler as long as `hideSidebarToggler` is `true`. This causes a bug in medium screens where the toggler gets hidden (it shouldnt). Now, toggler isn't hidden anymore in medium screens.

Before:
![Screenshot 2023-06-13 at 18 21 22](https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/5e56a519-af0d-4e2c-94d6-2c2ef3275945)
(no sidebar toggler)

After:

![Screenshot 2023-06-13 at 18 21 05](https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/b04ac424-3fc6-4ca2-8c73-e27d2ed69999)